quickwit/quickwit-index-management/src/index.rs

```diff
@@ -193,6 +197,76 @@ impl IndexService {
         Ok(deleted_splits)
     }
 
+    /// Deletes the indexes specified with `index_id_patterns`.
+    /// This is a wrapper of delete_index, and support index delete with index pattern
+    ///
+    /// * `index_id_patterns` - The targeted index ID patterns.
+    /// * `dry_run` - Should this only return a list of affected files without performing deletion.
+    pub async fn delete_indexes(
+        &self,
+        index_id_patterns: Vec<String>,
+        dry_run: bool,
+    ) -> Result<Vec<SplitInfo>, IndexServiceError> {
+        let indexes_metadata = self
+            .metastore
+            .list_indexes_metadatas(ListIndexesQuery::IndexIdPatterns(index_id_patterns.clone()))
+            .await?;
+
+        if indexes_metadata.is_empty() {
+            return Err(IndexServiceError::Metastore(MetastoreError::NotFound(
+                EntityKind::Indexes {
+                    index_ids: index_id_patterns,
+                },
+            )));
+        }
+        let index_ids = indexes_metadata
+            .iter()
+            .map(|index_metadata| index_metadata.index_id())
+            .collect_vec();
+        info!(index_ids = ?index_ids);
```

Comment:
this logs is not clear. Something like `info!(index_ids = ?index_ids, "delete indexes")` would be better.

Reply:
Let's just use pretty sample for this:

```rust
use quickwit_common::PrettySample;

info!("deleting {} indexes", PrettySample::new(&index_ids, 5)
```